src/download.test.ts

```diff
@@ -114,9 +120,15 @@ describe("getSkylinkUrl", () => {
   });
 
   it("should return correctly formed URLs with forced download and path", () => {
-    expect(client.getSkylinkUrl(skylink, { download: true, path: "foo?bar" })).toEqual(
-      `${expectedUrl}/foo%3Fbar${attachment}`
-    );
+    const url = client.getSkylinkUrl(skylink, { download: true, path: "foo?bar" });
+
+    expect(url).toEqual(`${expectedUrl}/foo%3Fbar${attachment}`);
+  });
+
+  it("should return correctly formed URLs with no-response-metadata set", () => {
```

Comment:
properly also want to test a case where there's multiple params added to the query string?